docs/dev/Dockerfile

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) 2012-2016 Codenvy, S.A.

```

Comment:
what is the purpose of that dockerfile ?
now Dockerfile are in /dockerfiles folder